### PR TITLE
feat: added tooltip to CustomCard

### PR DESCRIPTION
**Pull request by quyet-v**
# Description

[comment]: # 'Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.'

Added a tool tip to the CustomCard component. Used by passing tool tip to props.

Fixes (#751)

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [ ] New feature (non-breaking change which adds functionality)

## Screenshots

[comment]: # 'Please include screenshots of your changes if relevant.'

image

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings

### CODE REVIEW

File: src/components/common/CustomCard.js [hunk 1]
```diff
@@ -51,9 +55,26 @@ function CustomCard(props) {
...
+        <Typography variant="body1" color="darkGrey.main"
+          sx={{
+            display: "inline-block"
+          }}
+        >
           {title}
         </Typography>
+        
+        {tooltip && ( 
+          <Tooltip title="tooltip">
```
Comment (RubenSmn) [inline src/components/common/CustomCard.js:67]: Could you make the title dynamic? So we can pass in the text from the tooltip argument

**Review by RubenSmn — CHANGES_REQUESTED** [review]

**Comment by RubenSmn** [comment]
Looks really great! Could you also remove the unused imports?

**Comment by quyet-v** [comment]
On it, thanks for the feedback :)

**Review by RubenSmn — APPROVED** [review]

**Comment by RubenSmn** [comment]
Could you fix the unit test errors?

**Comment by RubenSmn** [comment]
@quyet-v thanks great work!

**Pull request merged**